flux_led/__main__.py

```diff
@@ -676,28 +725,33 @@ def __str__(self):
           mode_str += str(_r) + ","
         return "{} [{}]".format(power_str, mode_str)
 
-    def turnOn(self, retry=2):
-        self._is_on = True
-        msg = bytearray([0x71, 0x23, 0x0f])
+    def turnOn(self, retry=2, turn_on = True):
```

Comment:
I would suggest move this into a setState function (or another name):
To avoid extra parameter turn_on in this function. I think turn_on() should only be called when you want to turn on a light.